src/models/PromoBanner.ts

```diff
@@ -0,0 +1,26 @@
+import * as Sequelize from "sequelize";
+import { sequelize } from "./index";
+
+export interface PromoBannerAttributes {
+  image: string;
+  url?: string;
```

Comment:
Could we maybe rename the attributes to `imageURL` and `linkURL` or something similar? It took me a minute to realize what `image` meant and how it was different from `url`.